secure-biometric-interface-integrator/lib/standardConstant.js

```diff
@@ -5,6 +5,13 @@ const states = {
   AUTHENTICATING: "AUTHENTICATING",
 };
 
+const ErrorCode = {
+  DEVICE_NOT_FOUND: "device_not_found_msg",
+  BIOMETRIC_CAPTURE_FAILED: "biometric_capture_failed_msg",
+  CAPTURE_TIMEOUT: "capture_timeout",
```

Comment:
capture_timeout error code is not required

Reply:
fix in this commit [5ebb03c](https://github.com/mosip/esignet-plugins/pull/31/commits/5ebb03c7250691483cd36883d9cac477a1431126)